composer.json

```diff
@@ -36,5 +38,16 @@
     "psr-4": {
       "Elasticsearch\\Tests\\":      "tests/Elasticsearch/Tests/"
     }
+  },
+  "config": {
+    "sort-packages": true
```

Comment:
this ensures that future changes to `composer.json` are sorted automatically (when calling `composer require`